README.rst

```diff
@@ -77,6 +77,29 @@ Remember to add a ``SECRET_KEY`` setting!
     python manage.py index
 
 
+Static Files
+------------
+
+Static files are stored in `sitemedia/` and bundled by Webpack. The `webpack-bundle-tracker` plugin generates a JSON manifest file listing the name and location of bundled files. This file — `webpack-stats.json` in production/QA/test and `webpack-stats-dev.json` for local development — is read by Django using `django-webpack-loader` so that the relevant files can be included in the template's `<head>`.
+
+Bundled files and the production JSON manifest are checked into git and will be collected by Django's `collectstatic` command. They should be recompiled and any changes should be committed as part of a release. To recompile bundles for production::
+
+    npm run build
+
+When actively developing stylesheets and scripts, you can instead run a development Webpack server with live reload capability. To do this, first uncomment or add the line to your `local_settings.py` that tells Django to read from the development Webpack manifest instead::
+
+    WEBPACK_LOADER["DEFAULT"]["STATS_FILE"] = BASE_DIR.parent / "sitemedia" / "webpack-stats-dev.json"
```

Comment:
this should just be `BASE_DIR`, not `BASE_DIR.parent`

Reply:
I agree, but something super weird is happening with importing that variable — it has different values in different settings files (for me, anyway). Would love to pair on it so I can show you at some point; suspect it has to do with how `django-split-settings` works.

Reply:
Since `local_settings.py` is in `./geniza`, isn't `BASE_DIR` here `./geniza`? and since `sitemedia` is at the top of the repo, you'd have to go up one?